src/en/mangabtt/src/eu/kanade/tachiyomi/extension/en/mangabtt/MangaBTT.kt

```diff
@@ -0,0 +1,261 @@
+        }.build()
+
+        val postBody = FormBody.Builder()
+            .add("StoryID", manga.url.substringAfterLast("-"))
+            .build()
+
+        return POST("$baseUrl/Story/ListChapterByStoryID", postHeaders, postBody)
+    }
+
+    override fun chapterListSelector() = "ul > li:not(.heading)"
+
+    override fun chapterFromElement(element: Element): SChapter = SChapter.create().apply {
+        element.selectFirst(".col-xs-4")?.also {
+            date_upload = it.text().trim().parseRelativeDate()
+        }
+        element.selectFirst("a")!!.run {
+            name = text().trim()
+            setUrlWithoutDomain(attr("href"))
+        }
+    }
+
+    // From OppaiStream
+    private fun String.parseRelativeDate(): Long {
+        val now = Calendar.getInstance().apply {
+            set(Calendar.HOUR_OF_DAY, 0)
+            set(Calendar.MINUTE, 0)
+            set(Calendar.SECOND, 0)
+            set(Calendar.MILLISECOND, 0)
+        }
+
+        var parsedDate = 0L
+        val relativeDate = this.split(" ").firstOrNull()
+            ?.replace("one", "1")
+            ?.replace("a", "1")
+            ?.toIntOrNull()
+            ?: return 0L
+
+        when {
+            // parse: 30 seconds ago
+            "second" in this -> {
+                parsedDate = now.apply { add(Calendar.SECOND, -relativeDate) }.timeInMillis
+            }
+            // parses: "42 minutes ago"
+            "minute" in this -> {
+                parsedDate = now.apply { add(Calendar.MINUTE, -relativeDate) }.timeInMillis
+            }
+            // parses: "1 hour ago" and "2 hours ago"
+            "hour" in this -> {
+                parsedDate = now.apply { add(Calendar.HOUR, -relativeDate) }.timeInMillis
+            }
+            // parses: "2 days ago"
+            "day" in this -> {
+                parsedDate = now.apply { add(Calendar.DAY_OF_YEAR, -relativeDate) }.timeInMillis
+            }
+            // parses: "2 weeks ago"
+            "week" in this -> {
+                parsedDate = now.apply { add(Calendar.WEEK_OF_YEAR, -relativeDate) }.timeInMillis
+            }
+            // parses: "2 months ago"
+            "month" in this -> {
+                parsedDate = now.apply { add(Calendar.MONTH, -relativeDate) }.timeInMillis
+            }
+            // parse: "2 years ago"
+            "year" in this -> {
+                parsedDate = now.apply { add(Calendar.YEAR, -relativeDate) }.timeInMillis
+            }
+        }
+        return parsedDate
+    }
+
+    // =============================== Pages ================================
+
+    override fun pageListParse(document: Document): List<Page> {
+        return document.select(".reading-detail > .page-chapter").map { page ->
+            val img = page.selectFirst("img[data-index]")!!
+            val index = img.attr("data-index").toInt()
+            val url = img.imgAttr()
+            Page(index, document.location(), url)
```

Comment:
`url` doesn't seem to be used in `imageRequest()` so you should be able to omit it

```suggestion
            Page(index, imageUrl = url)
```